<?php

use Symfony\Component\DependencyInjection\Argument\RewindableGenerator;
use Symfony\Component\DependencyInjection\Exception\RuntimeException;

// This file has been auto-generated by the Symfony Dependency Injection Component for internal use.
// Returns the private '.service_locator.TW_mS5m' shared service.

return $this->privates['.service_locator.TW_mS5m'] = new \Symfony\Component\DependencyInjection\Argument\ServiceLocator($this->getService, [
    'answerRepository' => ['privates', 'App\\Repository\\AnswerRepository', 'getAnswerRepositoryService.php', true],
    'scoreCalculator' => ['privates', 'App\\Service\\ScoreCalculator', 'getScoreCalculatorService.php', true],
    'surveyAnswer' => ['privates', '.errored..service_locator.TW_mS5m.App\\Entity\\SurveyAnswer', NULL, 'Cannot autowire service "debug.argument_resolver.not_tagged_controller.inner": it references class "App\\Entity\\SurveyAnswer" but no such service exists.'],
    'translationManager' => ['privates', 'App\\Service\\TranslationManager', 'getTranslationManagerService.php', true],
], [
    'answerRepository' => 'App\\Repository\\AnswerRepository',
    'scoreCalculator' => 'App\\Service\\ScoreCalculator',
    'surveyAnswer' => 'App\\Entity\\SurveyAnswer',
    'translationManager' => 'App\\Service\\TranslationManager',
]);
